### PR TITLE
Update process-auditing.md

### PR DESCRIPTION
**Pull request by magicansk**
Add MD syntax.

### CODE REVIEW

File: docs/wiki/deployment/process-auditing.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Process auditing is available in osquery, but requires additional configuration. It uses the same event-based architecture as the [File Integrity Monitoring (FIM)](../deployment/file-integrity-monitoring.md). To read more about how event-based tables are created and designed, check out the osquery [Table Pubsub Framework](../development/pubsub-framework.md). On all supported platforms, process events are abstracted into [`process_events`](https://osquery.io/schema/#process_events). This table abstracts the basic details about process creation. Process state changes and destruction is not yet represented, nor planned.
 
 To collect process events add a query like:
-```
+```sql
```
Comment (theopolis) [inline docs/wiki/deployment/process-auditing.md]: Are there trailing spaces here?

Comment (magicansk) [reply docs/wiki/deployment/process-auditing.md]: Fixed. Thanks!

**Comment by obelisk** [comment]
Can we merge all these pull requests into one?

**Comment by theopolis** [comment]
See: https://github.com/facebook/osquery/commit/8bf2d9e592b22aa38ca7b60c7c56a77a8da2c911

**Comment by magicansk** [comment]
How to merge all the docs PR into one?

**Comment by theopolis** [comment]
@magicansk I took care of it, I pulled each of your branches into a new branch, then squashed them `git rebase -i HEAD~3` and placed that commit onto `master`. I don't like doing it manually, but it's better than 3 very small related commits.

**Comment by magicansk** [comment]
Yes, 
I was going to merge all three docs that I committed into one then PR,  
but didn't figure out how it works. Thanks for pulled each my branches into one.

**Pull request closed without merging**